create_infotexts.py

```diff
@@ -494,13 +489,13 @@ def process_region_addition_to_description(region_str, country_str):
 
         return region_addition
 
-    def add_catch_all_category(self, item):
+    def add_catch_all_category(self):
         """"
         Check if there are any content cats added and add generic content category to image.
         
         Populate self.content_cat with commons category, if present.
         """
-        if not self.content_cats:
+        if len(self.content_cats) == 0:
```

Comment:
I changed because I tried this:
    In [25]: l = []

    In [26]: l == True
    Out[26]: False

    In [27]: l is True
    Out[27]: False

    In [28]: l == False
    Out[28]: False

    In [29]: l is False
    Out[29]: False
However, quirky enough the following works:
 ```
 In [30]: if l:
              print("l is True?")
          else:
              print("nah, l is False.")
nah, l is False.
```